lib/Field/TextField.php

```diff
@@ -1,17 +1,25 @@
 <?php
 
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace Symfony\Cmf\Component\ContentType\Field;
 
-use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Cmf\Component\ContentType\FieldInterface;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
-use Symfony\Cmf\Component\ContentType\View\ScalarView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class TextField implements FieldInterface
 {
     public function getViewType()
     {
-        return ScalarView::class;
+        return 'scalar';
```

Comment:
Note that the FQN approach should not be used in the mapping of types to the entity, i.e.

```yaml
Fqn\To\Class:
    properties:
        title:
            type: Symfony\Cmf\Component\ContentType\Field\TextType
```

Seems to me to be unacceptable. But in the case of the FieldInterface instances themselves, it makes sense I think (if only to be consistent with the Form framework).